### PR TITLE
Update 0500-10-25-tm-user_en.md

### PR DESCRIPTION
**Pull request by vdgwebsites**
A few minor additions.

### CODE REVIEW

**Comment by michael63-osm** [comment]
I commented your first change - your opinion please

**Comment by vdgwebsites** [comment]
It seems that this staging version of learnosm is not in sync with the live version learnosm.org

Sorry, i don't see the comment you mentioned.

gh-116

**Comment by michael63-osm** [comment]
sorry, my comment had not been posted until now - my fault with the interface

**Comment by vdgwebsites** [comment]
Closed.

File: _posts/en/0500-10-25-tm-user_en.md [hunk 1]
```diff
@@ -212,6 +212,8 @@ Select this to automatically start the OpenStreetMap iD editor within the Task M
 
 The editor will load in a new window or tab. Potlatch will not automatically display the task bounding box at this stage.
 
+Attention: you may need to install a Flash player to use Potlatch.
```
[the remark this answers is not in the document]
Comment (vdgwebsites) [reply _posts/en/0500-10-25-tm-user_en.md:215]: Potlatch will continue as an adobe air app. 

https://www.systemed.net/potlatch/download/

**Pull request closed without merging**